### PR TITLE
config: General cleanup

### PR DESCRIPTION
**Pull request by jlucovsky**
[PREVIEW] The configuration section(s) in the documentation will be updated once the changes in `suricata.yaml.in` have been resolved.

Describe changes
- General cleanup of typos and grammar issues.

### CODE REVIEW

**Comment by catenacyber** [comment]
Nice work

File: suricata.yaml.in [hunk 1]
```diff
@@ -113,7 +113,7 @@ outputs:
 
       # Community Flow ID
       # Adds a 'community_id' field to EVE records. These are meant to give
-      # a records a predictable flow id that can be used to match records to
+      # records a predictable flow ID that can be used to match records to
       # output of other tools such as Bro.
```
Comment (jasonish) [inline suricata.yaml.in]: Should change this to Zeek while in here.

File: suricata.yaml.in [hunk 2]
```diff
@@ -971,12 +975,12 @@ asn1-max-frames: 256
 ## Run Options
 ##
 
-# Run suricata as user and group.
+# Run Suricata with a specific user- and group-id:
```
Comment (jasonish) [inline suricata.yaml.in]: The `user-` looks fine, probably better with `user` or `user-id`.

**Comment by jasonish** [comment]
Good stuff.

File: suricata.yaml.in (hunk 1 above)
Comment (victorjulien) [reply suricata.yaml.in]: Zeek (Bro) would be my suggestion.

**Comment by jlucovsky** [comment]
Continued in #4652

**Pull request closed without merging**